### PR TITLE
Edit dashboards 1

### PR DESCRIPTION
**Pull request by Partha607**

### CODE REVIEW

File: docs/features/coursemanagement/permissions.md [hunk 1]
```diff
@@ -2,5 +2,69 @@
...
 
-----------
-TODO add content
\ No newline at end of file
+Platform and course administrators can grant permissions to a user under the course administration menu for a specific course.  
+
+
+![Permission](../../img/course_admin_items/course_permission.png)  
+*Fig. Select the Course Permission from the drop-down list of Course Administration*
+
```
Comment (tstaubitz) [inline docs/features/coursemanagement/permissions.md:10]: This page should be much more abstract. 

I'd suggest sth. like the following:
"The platform offers a very flexible and highly configurable roles and rights management system.
This system comes in a default configuration and can be adjusted according to the needs of the partners separately per platform instance.

Roles can be platform wide or course wide. The permissions that are assigned to each group can be defined and configured very fine-grained. "

You can remove the rest.

File: docs/courseadministration/addcontent/learningunits/text.md [hunk 2]
```diff
@@ -26,9 +26,16 @@ To link a file for download:
 
 - Upload the file via the file upload drop zone next to the text input field.
 - Click on the add image icon and select the file that you have just uploaded.
-- In the text field: **remove** the "!" in front of the download link and add some meaningful text within the brackets
-
-Icon type:
+- In the text field: **remove** the ***!*** in front of the download link and add some meaningful text within the brackets followed by the link.
+- For the link, you can also add a number like footnote and list the link numbers at the bottom of your text in the following format:
```
[the remark this answers is not in the document]
Comment (Partha607) [reply docs/courseadministration/addcontent/learningunits/text.md:30]: I know it's not a footnote, I mentioned that it was "like a footnote" but now I realised that it was a bad example. 
I'll apply all the changes👍

File: docs/features/analytics/dashboards/videos.md [hunk 3]
```diff
@@ -2,5 +2,40 @@
...
+*Fig. Sample preview of the statistical details upon clicking **More Details** under the Actions column of a particular video item in a specific course*  
+
+The above figure illustrates the video item details based on the following parameters:
+
+* Change Speed - Number of times the speed of the video was changed
+* Seek Start - Number of times the video was played again from the beginning
+* Pause - Number of times the video was paused
+* Play - Number of times the video was played  
+
+The above illustration can be used to analyse the video item based on the parameters.
```
[the remark this answers is not in the document]
Comment (Partha607) [reply docs/features/analytics/dashboards/videos.md:41]: The parameters for this instance are the events - change speed, play, pause etc. Is it incorrect? I can call them "events"

[the remark this answers is not in the document]
Comment (tstaubitz) [reply docs/features/analytics/dashboards/videos.md:41]: yes, events is much better

**Comment by tstaubitz** [comment]
A newer version of these pages already exist or are currently created

**Pull request closed without merging**